publications/publications.bib

```diff
@@ -10,6 +10,15 @@ @INPROCEEDINGS{santos_barnes_lo_et_al_ieee_mass_2014
 	url = {https://ieeexplore.ieee.org/document/7035779},
 	keywords={signal to noise ratio, global positioning system, finite element analysis, wildlife, hardware, receivers, aircraft, autonomous aerial vehicles, directional antennas, radio tracking, signal processing, small unmanned aerial vehicle system, wildlife radio collar tracking, VHF signal, custom signal processing algorithms, small fixed-wing aircraft drone, wildlife telemetry, software-defined radio, digital signal processing},
 	abstract={This paper describes a low cost system for tracking wildlife that is equipped with radio collars. Currently, researchers have to physically go into the field with a directional antenna to try to pinpoint the VHF (very high frequency) signal originating from a wildlife tracking collar. Depending on the terrain, it could take an entire day to locate a single animal. To vastly improve upon this traditional approach, the system proposed here utilizes a small fixed-wing aircraft drone with a simple radio on-board, flying an automated mission. Received signal strength is recorded, and used to create a heat map that shows the collar's position. A prototype of this system was built using off-the-shelf hardware and custom signal processing algorithms. Initial field tests confirm the systems capabilities and its promise for wildlife tracking.}}
+
+
+@inproceedings{ayers2021reducing,
```

Comment:
Please follow the naming convention as set out in https://github.com/UCSD-E4E/website/blob/master/publications/README.md.

Replicated here:

Set the BibTeX key as follows: first three last names separated by underscores, followed by "et al" if more than three authors, followed by the abbreviated journal/conference/publisher, followed by the year. If more than one entry matches the same key, suffix this with "_2" for the second publication, "_3" for the third, and so on.